backend/src/main/java/org/tim/controllers/ReportController.java

```diff
@@ -0,0 +1,21 @@
+package org.tim.controllers;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.tim.services.ReportService;
+import static org.tim.constants.Mappings.*;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(API_VERSION + REPORT)
+public class ReportController {
+
+    private final ReportService reportService;
+//TODO: Do i get locales as strings here or Locale objects?
+    @GetMapping(path = GENERATE)
+    public ResponseEntity<FileSystemResource> generateExcelReport(@PathVariable Long projectId, @RequestParam String[] locales) {
```

Comment:
The variable created in this method should have the same name as the one defined in the GENERATE variable ->  GENERATE = "/generate/{id}". You should change this name (projectId -> id) or add information that you want to use it like that (@PathVariable("id") Long projectId). Without these changes, this endpoint will not work properly. Additionally, the word -> path is unnecessary. This is the default field for this annotation.